velox/core/PlanNode.cpp

```diff
@@ -822,12 +822,14 @@ UnnestNode::UnnestNode(
     std::vector<FieldAccessTypedExprPtr> unnestVariables,
     const std::vector<std::string>& unnestNames,
     const std::optional<std::string>& ordinalityName,
-    const PlanNodePtr& source)
+    const PlanNodePtr& source,
+    const bool outer)
```

Comment:
Hi @boneanxs, thanks for your work. Let's create an issue to describe the issue we met, and discuss the proposed solution with the community first.

Reply:
Hey, @rui-mo, raised an issue here: https://github.com/facebookincubator/velox/issues/9630

Reply:
Thank you.